src/Polymorph-Widgets/MorphicUIManager.class.st

```diff
@@ -648,11 +648,11 @@ MorphicUIManager >> showWaitCursorWhile: aBlock [
 { #category : #'ui process' }
 MorphicUIManager >> spawnNewProcess [
```

Comment:
For Pharo12 method neeeds to be copied to MorphicCoreUIManager (where it is duplicated)